git-pr-merge

```diff
@@ -41,14 +63,14 @@ title_prefix='✨ '
 delete_remote_branch='true'
 sleep_time_before_delete='15'
 
-prmerge::read_config() {
+common::read_config() {
   prmerge::_migrate_config
```

Comment:
also rename `prmerge::_get_config`. I originally left these because they are using the section `pr.merge` but I dont think that matters now you point it out.

Reply:
done